src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -611,6 +610,12 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
             network_policy=network_policy,
             load_balancer_sku=load_balancer_sku
         )
+    else:
+        if load_balancer_sku.lower() == "standard":
+            network_profile = ContainerServiceNetworkProfile(
+                network_plugin="kubenet",
+                load_balancer_sku=load_balancer_sku
```

Comment:
can we make it lower before passing to AKS RP? Although it works, it's a bit weird when customer see the data:

$ az aks show -g tonyxuRGEastUS2EUAP -n myAKSCluster3  | grep loadBalancerSku -C 4
The behavior of this command has been altered by the following extension: aks-preview
  "name": "myAKSCluster3",
  "networkProfile": {
    "dnsServiceIp": "10.0.0.10",
    "dockerBridgeCidr": "172.17.0.1/16",
    **"loadBalancerSku": "STANDard",**
    "networkPlugin": "kubenet",
    "networkPolicy": null,
    "podCidr": "10.244.0.0/16",
    "serviceCidr": "10.0.0.0/16"